src/app/lib/ledger.ts

```diff
@@ -17,6 +19,20 @@ export async function canAccessNavigatorUsb(): Promise<boolean> {
   return await isSupported()
 }
 
+export async function canAccessBle(): Promise<boolean> {
+  let isSupportedOnMobile = false
+  try {
+    // Returns always true on web
+    isSupportedOnMobile = await BleTransport.isEnabled()
+  } catch (ex) {
+    /* ignore */
+  }
+  // Scan depends on requestLEScan method, which is not available on the web(feature flag)
+  const isSupportedOnDesktop = Capacitor.isNativePlatform() || !!navigator?.bluetooth?.requestLEScan
+
+  return isSupportedOnMobile && isSupportedOnDesktop
```

Comment:
isSupportedOnDesktop is awkwardly named now

Maybe
```suggestion
  const hasBLE = await BleTransport.isEnabled().catch(() => false)
  // Scan depends on requestLEScan method, which is not available on the web(feature flag)
  const hasLEScan = Capacitor.isNativePlatform() || !!navigator?.bluetooth?.requestLEScan

  return hasBLE && hasLEScan
```